### PR TITLE
Add people and books

### PR DESCRIPTION
**Pull request by V0pe**
### PERSON AND BOOKS CREATION

In this project, I did the following to achieve the goal: 
- [ ] Add the classes
- [ ] setup the setter and getter
- [ ] Created the private and public method
- [ ] Inherit the child class from parent
-

### CODE REVIEW

File: teacher.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown')
+    super(age, name)
```
Comment (TedLivist) [inline teacher.rb]: - [ ] You are missing the `parent_permission` parameter for the `#initialize` and `#super` methods. Kindly add it.

File: person.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1...1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
```
Comment (TedLivist) [inline person.rb:11]: - [ ] **[OPTIONAL]** I think it would be nice to move these getters and setters to the top of the class for better organization.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @V0pe,

Good job so far!
✔️ Classes are in different files in keeping with Ruby convention
✔️ Good documentation
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 2 above)
Comment (V0pe) [reply person.rb:11]: @TedLivist Thanks , I will adjust accordingly

File: teacher.rb (hunk 1 above)
Comment (V0pe) [reply teacher.rb]: @TedLivist I have adjusted it. I felt it was unnecessary.

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @V0pe 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**